gdx/src/com/badlogic/gdx/utils/Queue.java

```diff
@@ -438,87 +468,108 @@ public boolean equalsIdentity (Object o) {
 			if (myValues[myIndex] != itsValues[itsIndex]) return false;
 			myIndex++;
 			itsIndex++;
-			if (myIndex == myBackingLength) myIndex = 0;
-			if (itsIndex == itsBackingLength) itsIndex = 0;
+			if (myIndex == myBackingLength)
+				myIndex = 0;
+			if (itsIndex == itsBackingLength)
+				itsIndex = 0;
 		}
 		return true;
 	}
 
+	/** The QueueIterator class is used to iterate through the queue.
```

Comment:
we know it's a class, an iterator, and it's used to iterate. Javadoc is not really necessary here.